include/spo2.h

```diff
@@ -0,0 +1,37 @@
+#ifndef __SPO2__
+#define __SPO2__
+
+#include <freertos/FreeRTOS.h>
+#include <freertos/task.h>
+
+#define SPO2_STACK_SIZE 0x1000
+
+
+/**
+ * struct _spo2_task - life-sensor task struct
+ * @tcb:	task related data
+ * @handle:	a handle to reference the task
+ * @stack:	the address of the tasks' stack
+ *
+ * This struct serves as a shortcut for operating with FreeRTOS tasks such
+ * that it includes necessary fields for managing threads.
+ */
+typedef struct {
+	StaticTask_t tcb;
+	TaskHandle_t handle;
+	StackType_t stack[SPO2_STACK_SIZE];
+} _spo2_task;
```

Comment:
no fan of names, which start with an underscore. Such names usually imply internal usage.